src/Microsoft.IdentityModel.Tokens/JsonWebKeySet.cs

```diff
@@ -123,68 +123,100 @@ public JsonWebKeySet(string json, JsonSerializerSettings jsonSerializerSettings)
         /// </summary>
         public IList<SecurityKey> GetSigningKeys()
         {
-            List<SecurityKey> keys = new List<SecurityKey>();
-            for (int i = 0; i < Keys.Count; i++)
-            {
-                JsonWebKey webKey = Keys[i];
+            var keys = new List<SecurityKey>();
```

Comment:
>keys [](start = 16, length = 4)

IMO, this should be renamed. Too close to the class level Keys. e.g. signingKeys

Reply:
I like _signingKeys_. I'm going to rename it.